sources/packages/web/src/components/students/StudentApplicationsVersion.vue

```diff
@@ -57,10 +79,19 @@ export default defineComponent({
       required: false,
     },
   },
-  setup() {
+  setup(props) {
     const { mapApplicationEditStatusForStudents } = useApplication();
     const { getISODateHourMinuteString } = useFormatters();
+    const filteredVersions = computed(() => {
+      if (!props.versions) {
+        return [];
+      }
+      return props.versions.filter((version) =>
+        APPLICATION_VERSIONS_STATUSES.includes(version.applicationEditStatus),
+      );
+    });
     return {
+      filteredVersions,
```

Comment:
I follow the logic but not the business requirement here which shows only original version and the ones edited post COE. What is the reason behind not including edits before COE? is that because they less importance to be shown here?

Reply:
They are also included. As the latest decision, we are basically removing versions that represent an in-progress change request, which means `ChangeInProgress` and `ChangePendingApproval`.